packages/pluginutils/package.json

```diff
@@ -79,13 +82,11 @@
     "require": "./dist/cjs/index.js",
     "import": "./dist/es/index.js"
   },
-  "module": "./dist/es/index.js",
   "nyc": {
     "extension": [
       ".js",
       ".ts"
     ]
   },
-  "type": "commonjs",
-  "types": "types/index.d.ts"
+  "type": "commonjs"
```

Comment:
could you explain this property? I'm not finding it in the typescript docs https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html

Reply:
This is a Node 13+ property to designate that files below this package file are NOT ES modules. It is the inverse of `"type": "module"`, but it is also equivalent to not specifying it at all https://nodejs.org/api/esm.html#esm_package_json_type_field
Not sure who added it originally.

Reply:
really appreciate the explanation. probably sounded like I was questioning it on the PR, poor choice of words. I was honestly hoping you knew.